### ISSUE TITLE
Debug data serialization improvements  

### ISSUE THREAD

**Issue by woodshop2300** [ackwell/attunement#13]
http://lua-users.org/wiki/TableSerialization
Offers a new number of ways to  serialize or pretty print a table.
I'm rather partial of the universal to string.
The only issue I see is the amount of vertical real estate it would take up on the council since we don't have that many lines and no scrolling.

**Comment by ackwell** [comment]
Might be worth holding off on this until next Aetherflow build. Once we've got D2D support, I'm planning on adding scrolling functionality to the console, so printing out large tables will be less of an issue vertical-space-wise.

**Comment by woodshop2300** [comment]
True, not to mention it would look like crap w/o a real mono space font
